src/main/java/ca/nrc/cadc/nameresolver/NRServlet.java

```diff
@@ -587,12 +624,13 @@ private void connectChannel(Selector selector, SocketChannel channel, SelectionK
                 final URL url = (URL) key.attachment();
                 channel.write(encoder.encode(CharBuffer.wrap("GET " + url.getPath() + "?" + url.getQuery()
                                                                  + "  HTTP/1.0\r\n\r\n")));
-                LOGGER.debug("  connected redirect channel: " + host);
+                LOGGER.debug(host + " -  connected redirect channel");
             } else {
                 final Service service = Service.valueOfFromHost(host);
                 final String connectString = service.getConnectString(target);
-                channel.write(encoder.encode(CharBuffer.wrap(connectString)));
-                LOGGER.debug(String.format("Connecting to '%s' with '%s'", host, connectString));
+                ByteBuffer encoded = encoder.encode(CharBuffer.wrap(connectString));
+                long bytes = channel.write(encoded);
```

Comment:
The `bytes` variable is not used.  Remove it.

Reply:
Leftover debug code, removed.

Reply:
Missed the start of the path for those files, removed.